.github/keylabeler.yml

```diff
@@ -9,21 +9,14 @@ caseSensitive: false
 
 # Explicit keyword mappings to labels. Form of match:label. Required.
 labelMappings:
-    "[ATMOS]": "Feature: Atmospherics"
-    "[AUDIO]": "Feature: Audio"
-    "[CONSTRUCTION]": "Feature: Construction"
-    "[ENTITIES]": "Feature: Entities"
-    "[ENTITY]": "Feature: Entities"
-    "[ENTITY AI]": "Feature: Entity AI"
-    "[EVENTS]": "Feature: Events"
-    "[EVENT]": "Feature: Events"
-    "[INTERACTION]": "Feature: Interaction"
-    "[MEDICAL]": "Feature: Medical"
-    "[PHYSICS]": "Feature: Physics"
-    "[POWER]": "Feature: Power"
-    "[SPRITES]": "Feature: Sprites"
-    "[SPRITE]": "Feature: Sprites"
-    "[UI]": "Feature: UI"
+    "[ATMOS]": "Area: Atmos"
+    "[BODY]": "Area: Body System"
```

Comment:
`[BODY SYSTEM]` is better in my opinion, but I think you can have duplicates so both is best if you ask me.

Reply:
Done, both are valid now.